python/GafferSceneUI/LightEditor.py

```diff
@@ -390,12 +390,18 @@ def __editSelectedCells( self, pathListing, quickBoolean = True ) :
 	def __toggleBoolean( self, inspectors, inspections ) :
 
 		plugs = [ i.acquireEdit() for i in inspections ]
-		# Make sure all the plugs are either a BoolPlug or contain a BoolPlug
+		# Make sure all the plugs either contain, or are themselves a BoolPlug or can be edited
+		# by `SetMembershipInspector.editSetMembership()`
 		if not all (
 			(
 				isinstance( p, ( Gaffer.TweakPlug, Gaffer.NameValuePlug ) ) and
 				isinstance( p["value"], Gaffer.BoolPlug )
-			) or isinstance( p, Gaffer.BoolPlug ) for p in plugs
+			) or (
+				isinstance( p, ( Gaffer.BoolPlug, Gaffer.Spreadsheet.RowPlug ) )
+			) or (
+				isinstance( p.parent(), ( GafferScene.ObjectSource ) )
+			)
+			for p in plugs
```

Comment:
Yes, I wasn't terribly happy about that, and since we have our list of inspectors readily available, maybe it's better to check against that? I did that in https://github.com/GafferHQ/gaffer/pull/5108/commits/6e7d2827fef2ee9c4bd57379c7fdbf8b64b47159 and I think it looks better.